### PR TITLE
Reformat code in `docs-search` using `purs-tidy`

### PR DESCRIPTION
**Pull request by CharlesTaylor7**
### Description of the change

- Reformats the `docs-search` package with purs-tidy. 
- Applies a CI check for the code format.

### CODE REVIEW

**Review by f-f — APPROVED** [review]
Amazing, thank you!

**Comment by f-f** [comment]
@CharlesTaylor7 added you as a collaborator so you can merge things once anyone approves 🙂

**Comment by CharlesTaylor7** [comment]
@f-f This is what it looks like for me. I think, I would need write access for that.

Screen Shot 2023-10-01 at 3 56 52 PM

**Pull request merged**
